src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -42,24 +47,30 @@ function NameAndPicture({ navigation }: Props) {
 
   // CB TEMPORARY HOTFIX: Pinging Komenci endpoint to ensure availability
   const asyncKomenciReadiness = useAsyncKomenciReadiness()
-
+  const showGuidedOnboarding = useSelector(showGuidedOnboardingSelector)
   const createAccountCopyTestType = useSelector(createAccountCopyTestTypeSelector)
 
   useLayoutEffect(() => {
     navigation.setOptions({
-      headerTitle: () => (
-        <HeaderTitleWithSubtitle
-          title={t(
-            choseToRestoreAccount
-              ? 'restoreAccount'
-              : createAccountCopyTestType === CreateAccountCopyTestType.Wallet ||
-                createAccountCopyTestType === CreateAccountCopyTestType.AlreadyHaveWallet
-              ? 'createProfile'
-              : 'createAccount'
-          )}
-          subTitle={t('registrationSteps', { step, totalSteps })}
-        />
-      ),
+      headerTitle: () => {
+        let pageTitleTranslationKey
+        if (showGuidedOnboarding) {
+          pageTitleTranslationKey = 'name'
+        } else {
+          pageTitleTranslationKey = choseToRestoreAccount
+            ? 'restoreAccount'
+            : createAccountCopyTestType === CreateAccountCopyTestType.Wallet ||
+              createAccountCopyTestType === CreateAccountCopyTestType.AlreadyHaveWallet
+            ? 'createProfile'
+            : 'createAccount'
```

Comment:
why would all of this logic not apply if someone was in the guided onboarding "on" group? Seems like we'd still want them to see these header titles if they were restoring their account, already had a wallet etc. Maybe the only difference is the case where `choseToRestoreAccount`, `createAccountCopyTestType === CreateAccountCopyTestType.Wallet`, and `createAccountCopyTestType === CreateAccountCopyTestType.AlreadyHaveWallet` all evaluate to false but `showGuidedOnboarding` is true?

Reply:
I believe because either restoring or new wallet would share the "name" title because both were considered onboarding and design wanted them to have this title when experiment is on, since the umbrella condition.